src/apps/inventory/views.py

```diff
@@ -41,7 +41,18 @@ class ProductBySellerListView(LoginRequiredMixin, ListView):
 
     def get_queryset(self):
         if self.request.user.is_authenticated:
-            return Product.objects.filter(seller=self.request.user)
+            return Product.objects.filter(seller=self.request.user, is_active=True)
+        return Product.objects.none()
```

Comment:
i think we can get rid of if-statement due to LoginRequiredMixin

leave only this
    def get_queryset(self):
        return Product.objects.filter(seller=self.request.user, is_active=True)